### ISSUE TITLE
Create mockup flows for user scenarios

### ISSUE THREAD

**Issue by umitcan07** [bounswe/bounswe2024group11#68]
# 🧱 Description

We should have a visual demonstration of the user scenarios as a part of the project plan.

## 📋 Acceptance Criteria

- [x] Every user scenario has a clear mockup in the respective Wiki pages.

**Comment by umitcan07** [comment]
I have designed the screens with @Meminseeker (6 hours of sprint indeed!), he will order them correctly, make minor adjustments and add them into the respective pages.

**Comment by Meminseeker** [comment]
It was a very tiring job. And now, we have those mock-up screens in our wiki. Check the scenario links, they are under Mock-ups section:

[Scenario 1](https://github.com/bounswe/bounswe2024group11/wiki/Scenario-1)
[Scenario 2](https://github.com/bounswe/bounswe2024group11/wiki/Scenario-2)
[Scenario 3](https://github.com/bounswe/bounswe2024group11/wiki/Scenario-3)
[Scenario 4](https://github.com/bounswe/bounswe2024group11/wiki/Scenario-4)
[Scenario 5](https://github.com/bounswe/bounswe2024group11/wiki/Scenario-5)

**Comment by ceydanursen** [comment]
I reviewed the mockups and all of them seem great and designed according to the scenarios we have. I just noticed one minor thing that should be changed.
**Scenario 4** has one action as **"He makes a comment."** For this part, there are two different mockups: one for liking & writing comment, the other one for comment parts. These can be shown in only one step under the liking & writing comment part.
